superset-frontend/cypress-base/cypress/e2e/dashboard/editmode.test.ts

```diff
@@ -42,15 +42,29 @@ function openProperties() {
     cy.getBySel('header-actions-menu')
       .contains('Edit properties')
       .click({ force: true });
-    cy.wait(500);
+    cy.get('.ant-modal-body').should('be.visible');
```

Comment:
could add a `data-test` attribute to select if that's considered more stable than the antD class name.